addons/pos_self_order/static/src/app/pages/confirmation_page/confirmation_page.js

```diff
@@ -60,8 +68,8 @@ export class ConfirmationPage extends Component {
             clearTimeout(this.defaultTimeout);
         });
 
-        onWillStart(() => {
-            this.initOrder();
+        onWillStart(async () => {
+            await this.initOrder();
```

Comment:
Carefull with this kind of stuff, if the network is slow and the request take more than 5 seconds the onWillStart will be drop

Reply:
Then, the initOrder shouldn't be onWillStart, it should be on onMounted.